src/test/java/car/CarListTest.java

```diff
@@ -1,10 +1,38 @@
 package car;
 
+import exceptions.CarException;
 import org.junit.jupiter.api.Test;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
 public class CarListTest {
 
+    @Test
+    public void addCar_validCarObject() {
+        Car car = new Car("ABC", "DEF", 180);
```

Comment:
maybe use Car 1, Car 2, and "Honda Civic", "Audi A4", etc as model, "ABC123","XYZ66" etc as license plate